docs/src/main/sphinx/release/release-408.md

```diff
@@ -0,0 +1,71 @@
+# Release 408 (22 Feb 2023)
+
+## General
+
+* Fix query failure when performing a remote partitioning exchange on a
+  [structural type](structural-data-types) that contains `NULL` elements. ({issue}`16140`)
+
+## Security
+
+* Deprecate the `http-server.authentication.oauth2.groups-field` configuration 
+  property by renaming it to
+  `deprecated.http-server.authentication.oauth2.groups-field`. ({issue}`15669`)
+
+## CLI
+
+* Add `AUTO` output format which switches from `ALIGNED` to `VERTICAL` if
+  the output doesn't fit the current terminal. ({issue}`12208`)
+
+## BigQuery connector
+
+* Add support for writing `DECIMAL` types to BigQuery. ({issue}`16145`)
+
+## Delta Lake connector
+
+* Add support for creating tables with the Delta Lake `change_data_feed_enabled`
+  table property. ({issue}`16129`)
+* Improve query performance on tables that Trino has written to with `INSERT`. ({issue}`16026`)
+* Improve performance of reading [structural types](structural-data-types) from
+  Parquet files. This optimization can be disabled with the
+  `parquet_optimized_nested_reader_enabled` catalog session property or the
+  `parquet.optimized-nested-reader.enabled` catalog configuration property. ({issue}`16177`)
+* Retry dropping Delta tables registered in the Glue catalog to avoid failures
+  due to concurrent modifications. ({issue}`13199`)
+* Fix inaccurate change data feed entries for `MERGE` queries. ({issue}`16127`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Hive connector
+
+* Retry dropping Delta tables registered in the Glue catalog to avoid failures
+  due to concurrent modifications. ({issue}`13199`)
+* Remove support for the deprecated `hive-hadoop2` connector name. ({issue}`16166`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Hudi connector
+
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Iceberg connector
+
+* Reduce memory usage when reading `$files` system tables. ({issue}`15991`)
+* Require the `iceberg.jdbc-catalog.driver-class` configuration property to be
+  set to prevent a "driver not found" error after initialization. ({issue}`16196`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Ignite connector
+
+* Add [Ignite connector](/connector/ignite). ({issue}`8098`)
+
+## SingleStore connector
+
+* Remove support for the deprecated `memsql` connector name, requiring the
+  `connector.name` property to set to `singlestore`. ({issue}`16180`)
+
+## SQL Server connector
+
+* Add support for pushing down `=`, `!=` and `IN` predicates over case-sensitive 
```

Comment:
`<>` instead of `!=`. While Trino supports `!=` for convenience, the "official" operator is `<>`

Reply:
I had it as that, then Ashhar put it as `!=` in the issue. 😢